### PR TITLE
Add Vanity Trophies projects

### PR DESCRIPTION
**Pull request by ftpmorph**
For verification, each of the NFTs listed under these policy ID's has a link to the Twitter account from which I've just tweeted this:

https://twitter.com/vanitytrophies/status/1424319262022373376

Example, the latest NFT under the main Vanity Trophies project:

https://pool.pm/fb04e22a9196ac9ac1ca806394b291e7e46637aa6b66d697ceb4d24f.InCryptoWeTrust10

Policy ID is right there in URL, link to that Twitter is in metadata.

Thx :)

### CODE REVIEW

File: collections.json [hunk 1]
```diff
@@ -2001,5 +2001,52 @@
...
+        ]
+      },
+      {
+        "logoUrl": "https://raw.githubusercontent.com/AlexDochioiu/nft_craze_gallery_data/main/logos/PsychAlgodelic.png",
+        "collectionName": "PsychAlgodelic Unsigs",
+        "policies": [
+          "ab468cbd30dbfcbbf0e4db159d08ea910667afa49ad36f16675f26e7"
+        ]
+      }
+    ]
```
Comment (AlexDochioiu) [inline collections.json:2050]: Hello @ftpmorph  and thanks for this. There's only one thing I'd like you to address before I merge, please. Try to get all the logos to be at most 150kb (resizing them to around 100 x 100 pixels should do the trick).

Comment (ftpmorph) [reply collections.json:2050]: Will do now :)

File: collections.json [hunk 2]
```diff
@@ -2001,5 +2001,52 @@
...
+        "5c08ff27f5defa4f0895d5aab8cb7d8c11e68793e51f39936ec3a316",
+        "abe8d2d78042daa5bce6ae433ed7f9a75137aa2d1f4fbcecaee3b443",
+        "c5034cf4cc8458612913f3074b7982ed1bb0a4fc2bdb469e61d04436",
+        "be30b6efdbc0ebd80314906ab0f1df971c597fecca6abb4b98be807d",
+        "d17a17939616e8c84cd80cc51d036cadfce7e67fe9ad00d6aaa9b86f",
+        "fb04e22a9196ac9ac1ca806394b291e7e46637aa6b66d697ceb4d24f"
+        ]
+      },
+      {
+        "logoUrl": "https://raw.githubusercontent.com/AlexDochioiu/nft_craze_gallery_data/main/logos/CardanoRx.png",
```
Comment (AlexDochioiu) [inline collections.json]: The name of the file here is wrong. The actual file name is `CardanoRxThumb.jpg`

File: collections.json [hunk 3]
```diff
@@ -2001,5 +2001,52 @@
...
+        "collectionName": "Cardano Rx",
+        "policies": [
+          "54b95b60f404560cd87d31594c11005220d8c63f831045668972bb28"
+        ]
+      },
+      {
+        "logoUrl": "https://raw.githubusercontent.com/AlexDochioiu/nft_craze_gallery_data/main/logos/PsychAlgodelic.png",
```
Comment (AlexDochioiu) [inline collections.json]: The file name is wrong here as well.

Comment (ftpmorph) [reply collections.json]: Yeah I committed the changes separately, it should all match up now :)

**Comment by AlexDochioiu** [comment]
All looks good now! Thanks @ftpmorph

**Pull request merged**